app/main.py

```diff
@@ -1,3 +1,53 @@
-def shop_trip():
-    # write your code here
-    pass
+from datetime import datetime
+
+from app.trip_cost import calculate_trip_cost
+from app.file_opening import file_open
+
+
+def shop_trip() -> None:
+    user_data = file_open("config.json")
+    fuel_price = user_data["fuel_price"]
+    customers = user_data["customers"]
+    shops = user_data["shops"]
+    for customer in customers:
+        cheeper_shop = dict()
```

Comment:
The variable name `cheeper_shop` seems to be a typo. Consider renaming it to `cheaper_shop` for clarity and correctness.